man/systemd.net-naming-scheme.xml

```diff
@@ -369,6 +369,16 @@
           property.</para></listitem>
         </varlistentry>
 
+        <varlistentry>
+          <term><constant>v249</constant></term>
+
+          <listitem><para>PCI hotplug slot names on s390 consist of 8 hexadecimal digits, left-padded with
+          zeros. This can lead to inconsistencies, as the <varname>ID_NET_NAME_SLOT</varname> property will
+          be built for interfaces with slot names coincidentally consisting of decimal digits, while
+          interfaces with hex characters will be skipped. To avoid this, PCI slot names starting with a zero
```

Comment:
`with a zero` -> `with a zero padding`?

Reply:
`with zero padding`

Reply:
Actually, "with a zero" is good here, because it precisely describes the logic.

Reply:
I will probably change the patch to be more explicit (8 chars, zero-prefixed) as suggested by Zbigniew and will need to reword then.